src/model/rest/FileEntry.ts

```diff
@@ -41,6 +41,9 @@ export class FileEntry {
     @Nullable(String)
     public mediaType: string | null = null;
 
+    @Property()
+    public fileProtectionLevel: string;
```

Comment:
I think this should be a union string 

```ts
type prtectionlevel = "Encrypted" | "Password" | "None";
```
where you put the type, you decide what is best.

Reply:
Added to typeings.

Reply:
Adjusted